deploy/swrs/transform/function/load_fuel_mapping.sql

```diff
@@ -153,16 +153,14 @@ $function$
         INSERT INTO swrs_load.fuel_mapping (fuel_type, fuel_carbon_tax_details_id) VALUES ('Wood Waste', 84);
         INSERT INTO swrs_load.fuel_mapping (fuel_type, fuel_carbon_tax_details_id) VALUES ('Wood Waste (bone dry tonnes)', 84);
         INSERT INTO swrs_load.fuel_mapping (fuel_type, fuel_carbon_tax_details_id) VALUES ('Flared Natural Gas CO2', 85);
-        INSERT INTO swrs_load.fuel_mapping (fuel_type, fuel_carbon_tax_details_id) VALUES ('Flared Natural Gas CH4', 86);
-        INSERT INTO swrs_load.fuel_mapping (fuel_type, fuel_carbon_tax_details_id) VALUES ('Flared Natural Gas N2O', 87);
-        INSERT INTO swrs_load.fuel_mapping (fuel_type, fuel_carbon_tax_details_id) VALUES ('Vented Natural Gas', 88);
+        INSERT INTO swrs_load.fuel_mapping (fuel_type, fuel_carbon_tax_details_id) VALUES ('Vented Natural Gas CH4', 86);
```

Comment:
I guess we have lots of magic numbers in here. That would be a good tech debt card

Reply:
definitely, many of these load functions have magic numbers loaded in the fkey IDs